src/libraries/System.Drawing.Common/src/System/Drawing/Drawing2D/GraphicsPathIterator.cs

```diff
@@ -78,7 +78,7 @@ public int NextSubpath(out int startIndex, out int endIndex, out bool isClosed)
         public int NextSubpath(GraphicsPath path, out bool isClosed)
```

Comment:
Should it have been a `GraphicsPath?` here and in a similar case below? Both APIs work when `null` is passed.